packages/ui-app/src/Button/Divider.tsx

```diff
@@ -8,13 +8,15 @@ import React from 'react';
 
 import classes from '../util/classes';
 
-function ButtonDivider ({ className, style }: DividerProps) {
-  return (
-    <div
-      className={classes('ui button compact mini basic', className)}
-      style={style}
-    />
-  );
+// ({ className, style }: DividerProps) {
+export default class ButtonDivider extends React.Component<DividerProps> {
+  render () {
+    const { className, style } = this.props;
+    return (
```

Comment:
Probably could use a blank line between var declarations & return, doesn't cost anything and aids in readability.